constants.bal

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+const string BASE_URL = "https://people.googleapis.com/v1";
+# Holds the value for URL of refresh token end point
+public const string REFRESH_URL = "https://www.googleapis.com/oauth2/v4/token";
+const string EMPTY_STRING = "";
+const string SPACE = " ";
+const string SLASH = "/";
+const string COLON = ":";
+const string QUESTION_MARK = "?";
+const string AMBERSAND = "&";
+const string EQUAL = "=";
+# Constant for paths.
+const string CONTACT_GROUP_PATH = "contactGroups";
+const string BATCH_CONTACT_GROUP_PATH = ":batchGet?";
+const string LIST_PEOPLE_PATH = "people/me/connections?requestSyncToken=true&syncToken=MisAPB3nNAAAABIIvbPr36uh7wIQvbPr36uh7wJhJlAhhPtQDf5q8aaPtMVGOgw4NDkzNTA1ODUxNjY=&";
```

Comment:
Here a token is added in this string. Is this correct? 
Can u check about this?

Reply:
It is no longer needed. removed